tests/bluetooth/tester/src/btp/btp_cap.h

```diff
@@ -0,0 +1,166 @@
+/* btp_cap.h - Bluetooth tester headers */
+
+/*
+ * Copyright (c) 2023 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* CAP commands */
+#define BTP_CAP_READ_SUPPORTED_COMMANDS		0x01
+struct btp_cap_read_supported_commands_rp {
+	uint8_t data[0];
+} __packed;
+
+#define BTP_CAP_DISCOVER			0x02
+struct btp_cap_discover_cmd {
+	bt_addr_le_t address;
+} __packed;
+
+#define BTP_CAP_ORDERED_ACCESS			0x03
+struct btp_cap_ordered_access_cmd {
+	bt_addr_le_t address;
+} __packed;
+
+#define BTP_CAP_UNICAST_SETUP_STREAM		0x04
+struct btp_cap_unicast_setup_stream_cmd {
+	bt_addr_le_t address;
+	uint8_t ase_id;
+	uint8_t cis_id;
+	uint8_t cig_id;
+	uint8_t coding_format;
+	uint16_t vid;
+	uint16_t cid;
+	uint8_t sdu_interval[3];
+	uint8_t framing;
+	uint16_t max_sdu;
+	uint8_t retransmission_num;
+	uint16_t max_transport_latency;
+	uint8_t presentation_delay[3];
+	uint8_t cc_ltvs_len;
+	uint8_t metadata_ltvs_len;
+	uint8_t ltvs[0];
```

Comment:
Why do we have both cc and metadata LTV lenghts, but only 1 ltvs array?

Reply:
The cc and metadata LTVs are packed one after the other, so the `uint8_t ltvs[0]` can be treated as a pointer to this LTV chain.

Reply:
Ah. I guess that makes sense so you can use zero-length arrays like this.